src/web/make-report.rkt

```diff
@@ -61,9 +61,17 @@
     (for/sum ([t tests]) (or (table-row-start t) 0)))
   (define total-result
     (for/sum ([t tests]) (or (table-row-result t) 0)))
-  (define total-bits
-    (for/sum ([t tests])
-      (representation-total-bits (get-representation (table-row-precision t)))))
+  (define maximum-accuracy
+    (for/sum ([test (in-list tests)])
+      (representation-total-bits
+       (get-representation
+        (table-row-precision test)))))
+  (match-define (list (list _ initial-accuracy) frontier) merged-cost-accuracy)
+  (define speedup-at-initial-accuracy
+    ;; The cost of the point that minimizes distance from the initial-accuracy
```

Comment:
I don't think it'd change much but probably better to do fastest that is higher accuracy. Also I believe the points should be sorted, so I'm not sure we need `first`. We might even be able to just do `findf` / `for/first`.

Reply:
Ah okay you answered my first question below here. If it's sorted, we could start from the end and get the cost of the first point with an accuracy higher than the initial point. I'll try that really quick.

In terms of the `first`, it's actually there to just get the cost from the point that `argmin` returns, which is kind of confusing. That's why I had the `match` there originally, so perhaps I'll go add those `point-cost` and `point-accuracy` helpers you were talking about to clarify this.

Reply:
Ok. Not a high priority but we might as well do it now while it's fresh in your head.